### ISSUE TITLE
Remove deprecations in t.web.server.Session

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6514]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6514|
|Type|task|
|Created|2013-05-14 18:36:29Z|

The following is deprecated in `twisted.web.server.Session` since v9.0 and can be removed:

 * `Session.loopFactory`
 * `lifetime` param of `startCheckingExpiration`
 * `checkExpired`

Attachments:

* [ticket6514.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/414/414f7e0016269fd51f4662093f167f5e681ba078/810f9337010c49b5f16ff7dc8846f5ee1a3de176.patch) (1919 bytes) - added by canibanoglu on 2013-05-25 08:59:51Z - 
* [ticket6514.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/414/414f7e0016269fd51f4662093f167f5e681ba078/afdb06e3f30808548de42fc84af830f0e8d8e208.patch) (2109 bytes) - added by canibanoglu on 2013-05-25 09:04:02Z - Removed parts from docstrings relevant to the removed attributes

<details><summary>Searchable metadata</summary>

```
trac-id__6514 6514
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1368556589000000 1368556589000000
changetime__1370062651000000 1370062651000000
version__None None
owner__therve therve
cc__thijs cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="canibanoglu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|canibanoglu set owner to canibanoglu|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="canibanoglu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|canibanoglu set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="canibanoglu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|canibanoglu removed owner<br>canibanoglu set status to `new`|
|-|-|

Sorry for the double patch, I have realized that I forgot to remove the removed attributes/parameters from docstrings. All should be good now. Waiting for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve commented|
|-|-|

Looks OK. I also removed the tests and the imports, applying.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_7"></a>|@therve set status to `closed`|
|-|-|

(In [38581]) Remove deprecated API session related from twisted.web.server.

Author: canibanoglu
Reviewer: therve
Fixes: [#6514](https://github.com/twisted/twisted/issues/6514)
